cmd/ebrelayer/txs/parser.go

```diff
@@ -42,11 +44,24 @@ func ParsePayload(valAddr sdk.ValAddress, event *events.LockEvent) (ethbridgeTyp
 		return witnessClaim, errors.New("empty recipient address")
 	}
 
+	// Sender type casting (address.common -> string)
+	tokenContractAddress := ethbridgeTypes.NewEthereumAddress(event.TokenContractAddress.Hex())
+
+	// Symbol formatted to lowercase
+	symbol := strings.ToLower(event.Symbol)
+	if symbol == "eth" && event.TokenContractAddress != common.HexToAddress("0x0000000000000000000000000000000000000000") {
```

Comment:
it'd be cool to have an `IsAddressZero(address common.Address) bool`  utility function

Reply:
https://github.com/cosmos/peggy/pull/68/commits/236c190846edf84e30fb8eb38f721d27a9bae191, f1761f52c2356d4f0d147ffee673c4df43e68f69